content/posts/deprecate-config-files-without-sphinx-or-mkdocs-config.md

```diff
@@ -0,0 +1,41 @@
+title: Deprecation of configuration files without Sphinx or MkDocs configuration
+date: December 16, 2024
+description: Announcing the deprecation of configuration files without Sphinx or MkDocs configuration.
+category: Changelog
+tags: deprecation, configuration-file
+authors: Santos Gallegos
+status: published
+image: /images/deprecated-config-file-without-sphinx-or-mkdocs-config.jpg
+image_credit: Photo by <a href="https://unsplash.com/@jernejgraj?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Jernej Graj</a> on <a href="https://unsplash.com/photos/photo-of-seashore-during-golden-hour-n44KHcbp1_E?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>
+
+We are announcing the deprecation of configuration files that do not contain a explicit `sphinx` or `mkdocs` configuration.
```

Comment:
This seems like bad phrasing, because we support lots of configurations without those keys, especially with build.commands and new build.jobs syntax.

Reply:
build.commands yes, build.jobs still run sphinx commands. Anyway, what about "Deprecation of projects using Sphinx or MkDocs without an explicit configuration file"? Changed all the sentences to that.